my-react-app/src/Components/Async.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+function FirstOne(){
+    const arr = ['Mutarama', 'Gashyantare', 'Werurwe', 'Mata']
+    return(
+        <>
+        <p>The following are the first four month of the year</p> 
+        <ul style={{listStyle:'none'}}>{arr.map(ar =><li className='text-success'>{ar}</li>)}</ul>
+        </>
+        
+    )
+}
+
+  function Calculator() {
+    let sum = 245;
+    
+    return(
+        <div>
+            <p>Try this <strong>{sum}</strong> number  </p>
```

Comment:
For consistency purpose , you can use bootstrap styling for bolding than using HTML  tag `strong`